atlasdb-api/src/main/java/com/palantir/atlasdb/metrics/MetricPublicationFilter.java

```diff
@@ -23,4 +23,11 @@
     MetricPublicationFilter NEVER_PUBLISH = () -> false;
 
     boolean shouldPublish();
+
+    /**
+     * @return A label which is used to deduplicate filters; it need only be unique across filters on a given metric.
+     */
+    default String getLabel() {
+        return this.getClass().getName();
```

Comment:
This isn't used anywhere in AtlasDB itself - presumably it is exposed for users of Atlas?

Reply:
exposed for future proofing, more than anything else. if you want to internalize it, fine by em.